pkg/rtc/participant.go

```diff
@@ -1969,6 +1970,7 @@ func (p *ParticipantImpl) rtcpSendWorker() {
 	// read from rtcpChan
 	for pkts := range p.rtcpCh {
 		if pkts == nil {
+			p.params.Logger.Infow("exiting RTCP send worker")
```

Comment:
Seeing what I think is a case of RTCP worker getting shut down unexpectedly. So, logging.